linebot/emoji.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package linebot
+
+import (
+	"encoding/json"
+)
+
+// Emoji type
+type Emoji struct {
+	Index     int
+	ProductID string
+	EmojiID   string
+}
+
+// MarshalJSON method of Emoji
+func (s *Emoji) MarshalJSON() ([]byte, error) {
+	return json.Marshal(&struct {
```

Comment:
there's no added/removed fields from the struct to the JSON, I think no need proxy object

Reply:
Thank you, fixed already.